Userland/Libraries/LibWeb/HTML/Window.cpp

```diff
@@ -189,7 +189,7 @@ i32 Window::run_timer_initialization_steps(TimerHandler handler, i32 timeout, JS
     // 8. Assert: initiating script is not null, since this algorithm is always called from some script.
 
     // 9. Let task be a task that runs the following substeps:
-    auto task = [weak_window = make_weak_ptr<Window>(), handler = move(handler), timeout, arguments = move(arguments), repeat, id]() mutable {
+    auto task = [this, weak_window = make_weak_ptr<Window>(), handler = move(handler), timeout, arguments = move(arguments), repeat, id]() mutable {
```

Comment:
Woah wait, why are we capturing this here? We should be using our weakptr

Reply:
That is, remove the `this` capture, and fix any errors by calling `window->thing()`

Reply:
Ouch! So many ways to shoot yourself in the foot 😵‍💫